src/main/java/frc/robot/Robot.java

```diff
@@ -34,6 +34,7 @@ public void disabledExit() {}
 
   @Override
   public void autonomousInit() {
```

Comment:
opening bracket needs to be on the line below this. Remove extra white space on line 37